lms/templates/learner_dashboard/programs.html

```diff
@@ -20,6 +20,16 @@
 <%block name="pagetitle">${_("Programs")}</%block>
 
 <div class="program-list-wrapper">
+    % if not len(programs):
+        <div class="dashboard">
+        <section class="empty-dashboard-message">
+            <p>${_("It looks like you aren't yet enrolled in any XSeries Programs!")}</p>
+              <a class="find-xseries-programs" href=""><i class="action-xseries-icon" aria-hidden="true"></i>
```

Comment:
@marcotuts @jimabramson 
I have added check for empty list of programs on dashboard. what will be url for `View XSeries Programs`.